src/OpenTelemetry.Api/Internal/SemanticConventions.cs

```diff
@@ -110,5 +110,15 @@ internal static class SemanticConventions
         public const string AttributeExceptionType = "exception.type";
         public const string AttributeExceptionMessage = "exception.message";
         public const string AttributeExceptionStacktrace = "exception.stacktrace";
+
+        // Http v1.21.0 https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/trace/semantic_conventions/http.md
```

Comment:
FYI - v1.21.0 doesn't contain semantic convention changes https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/CHANGELOG.md#v1210-2023-05-09

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/17327289/1958537e-17ac-4d1e-9339-6e997c503f79)

Reply:
The **semantic-conventions** repo doesn't have any [tags ](https://github.com/open-telemetry/semantic-conventions/tags) yet so I can't link to a versioned copy of the Http document.


I discussed this with Utkarsh last week and he suggested using the v1.21.0 tag on the **opentelemetry-specification** repo.
The **opentelemetry-specification** repo does have tags and that version of the document does include all of the new attributes that we're adding in this PR.